docs/integrations/custom-media-app.md

```diff
@@ -0,0 +1,201 @@
+
+
+## <a id="prerequisites"></a> Prerequisites
+
+To integrate your media app with Metisa:
+
+1. **You must have a Metisa account.** Custom integrations are available to all Metisa users, whether you're on a Free or Paid tier. Sign up [here](https://askmetisa.com/create).
+2. **You must have basic technical know-how.** You will be fine if you're comfortable navigating files and inserting code blocks, and some light coding with guidance.
+3. **Your app must use a template rendering engine.** EJS, Liquid, Django are some examples. This installation rides on your app's template tags to obtain data to send to Metisa servers.
+
+On the last point, if you are a developer, you may be able to customize the way data is sent to Metisa  servers by fiddling with the [Metisa JavaScript SDK](https://github.com/altitudelabs/metisa-js).
+
+
+## <a id="concepts"></a> Key concepts
+
+Recommendations are made based on machine learning algorithms that feed on your app data like items (eg. articles), actions (eg. read), and users.
+
+We therefore need a way for your app to automatically send such data to Metisa, so that over time (3-7 days depending on site traffic), Metisa will learn enough about your users' individual preferences to start providing good recommendations that get more accurate with time.
+
+The way to achieve this is through inserting some JavaScript code into the page templates of your web app.
+
+
+## <a id="install"></a> Install
+
+> **TIP**: This section can be a little tricky.
+
+> We encourage you to refer to our [demo movie recommendations app](https://github.com/altitudelabs/metisa-js) to see an example implementation.
+
+There are two main steps to installing the scripts that enable custom integration.
+
+1. [Embed site-wide code](#embed-site-wide-code)
+2. [Embed template-specific code](#embed-template-code)
+
+#### <a id="embed-site-wide-code"></a> Step 1: Embed site-wide code snippets
+
+Metisa requires v1.5+ jQuery, a library used frequently by web developers. Please make sure jQuery is available.
+
+Add this to your base template file if your site uses an older version of jQuery or does not use it at all. (Note: adding this code block when your site already loads jQuery has a very slim chance of breaking your site due to differences between versions.)
+
+```html
+<script src="https://code.jquery.com/jquery-3.2.1.min.js"
+integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
+crossorigin="anonymous"></script>
+```
+
+> **TIP**: Your base template filename will vary depending on how your developer named it.
+
+> Common names are index and theme, with file extensions like .html, .erb, .ejs, .liquid (eg. index.ejs or theme.liquid)
+
+Once jQuery is available, paste the following script to the same file below jQuery. This will load the Metisa script that is needed to send and receive data to/from Metisa.
+
+```html
+<script src="https://metisa-sdk.s3-ap-southeast-1.amazonaws.com/js/v0.0/browser.js"></script>
```

Comment:
this should be metisa.js and needs to be updated on S3?

Reply:
👍 Ok will update here, shouldn't have let it slip!

I've already updated metisa-js/docs for `metisa.js`